stm32cube/stm32mp2xx/README

```diff
@@ -44,4 +44,7 @@ Patch List:
    - Added stm32cube/stm32mp2xx/drivers/include/stm32_assert.h
    - Removed unused stm32cube/stm32mp2xx/drivers/include/stm32_assert_template.h
 
+   * HAL LVDS module disabled
+   - Disabled HAL_LVDS_MODULE_ENABLED as LVDS HAL is not available yet
```

Comment:
If is direct copy paste of the Cube HAL as available on https://github.com/STMicroelectronics/STM32CubeMP2, then no need to document.
What should be documented are changes on top if it